CorsixTH/Lua/dialogs/resizables/file_browsers/save_game.lua

```diff
@@ -63,6 +63,14 @@ function UISaveGame:abortName()
   self.new_savegame_textbox.panel:setLabel(_S.save_game_window.new_save_game)
 end
 
+--! Updates the textbox to selected file
+--!param label (string) Selected file name
+function UISaveGame:updateTextbox(label)
+  local name = string.gsub(label,"%.sav","")
```

Comment:
``` lua
> label = "my.save"
> string.gsub(fname,"%.sav","")
mye
```
Also, some white space after the commas looks better.

Reply:
Fix: `string.gsub(fname, "%.sav$", "")`

Reply:
This should be a non-issue, see screenshots.
![image](https://user-images.githubusercontent.com/20030128/189078897-f7e59ae0-8294-44fd-9b0b-e66c6598b75d.png)
![image](https://user-images.githubusercontent.com/20030128/189078971-8d6b2008-ec33-450f-8efe-e4cdc3c9cf8a.png)

However, I'll add it in anyway.

Reply:
Chances is 100% without `$`, a `gsub` does global substitution if you allow it.